app/src/test/java/org/simple/clinic/summary/assignedfacility/AssignedFacilityInitTest.kt

```diff
@@ -0,0 +1,25 @@
+package org.simple.clinic.summary.assignedfacility
+
+import com.spotify.mobius.test.FirstMatchers.hasEffects
+import com.spotify.mobius.test.FirstMatchers.hasModel
+import com.spotify.mobius.test.InitSpec
+import com.spotify.mobius.test.InitSpec.assertThatFirst
+import org.junit.Test
+import java.util.UUID
+
+class AssignedFacilityInitTest {
+
+  @Test
+  fun `when screen is created, then load assigned facility`() {
```

Comment:
Should we also add a check when the screen is being restored to not load the assigned facility?